--- iproute2-4.14.1/configure.0111~	2017-11-14 12:00:48.708285970 +0100
+++ iproute2-4.14.1/configure	2017-11-14 12:01:37.776447098 +0100
@@ -315,7 +315,7 @@ int main(int argc, char **argv) {
 	return 0;
 }
 EOF
-    $CC -I$INCLUDE -o $TMPDIR/dbtest $TMPDIR/dbtest.c -ldb >/dev/null 2>&1
+    $CC REPLACE_HEADERS -o $TMPDIR/dbtest $TMPDIR/dbtest.c -ldb >/dev/null 2>&1
     if [ $? -eq 0 ]
     then
 	echo "HAVE_BERKELEY_DB:=y" >>$CONFIG
