diff -Naur iproute2-4.20.0/configure iproute2-4.20.0.tpg/configure
--- iproute2-4.20.0/configure	2019-01-07 18:24:02.000000000 +0000
+++ iproute2-4.20.0.tpg/configure	2019-01-31 14:33:07.407536279 +0000
@@ -276,7 +276,7 @@
 	return 0;
 }
 EOF
-    if $CC -I$INCLUDE -o $TMPDIR/dbtest $TMPDIR/dbtest.c -ldb >/dev/null 2>&1; then
+    if $CC REPLACE_HEADERS -o $TMPDIR/dbtest $TMPDIR/dbtest.c -ldb >/dev/null 2>&1; then
 	echo "HAVE_BERKELEY_DB:=y" >>$CONFIG
 	echo "yes"
     else
