--- iproute2-3.9.0/ip/ipxfrm.c.0005~	2013-06-20 01:46:19.294153421 +0200
+++ iproute2-3.9.0/ip/ipxfrm.c	2013-06-20 01:47:10.624511703 +0200
@@ -1275,6 +1275,7 @@ static int xfrm_selector_upspec_parse(st
 		case IPPROTO_UDP:
 		case IPPROTO_SCTP:
 		case IPPROTO_DCCP:
+		case IPPROTO_IP:	/* to allow shared SA for different protocols */
 			break;
 		default:
 			fprintf(stderr, "\"sport\" and \"dport\" are invalid with PROTO value \"%s\"\n", strxf_proto(sel->proto));
