diff -up iproute2-6.0.0/lib/ll_proto.c.1~ iproute2-6.0.0/lib/ll_proto.c
--- iproute2-6.0.0/lib/ll_proto.c.1~	2022-10-05 18:37:01.522724232 +0200
+++ iproute2-6.0.0/lib/ll_proto.c	2022-10-05 18:37:26.915913258 +0200
@@ -26,6 +26,10 @@
 #include "utils.h"
 #include "rt_names.h"
 
+/* missing define in PPC if_ether.h */
+#ifndef ETH_P_ECHO
+#define ETH_P_ECHO     0x0200
+#endif
 
 #define __PF(f,n) { ETH_P_##f, #n },
 
