diff -Nurp iproute2-3.2.0.orig/lib/ll_proto.c iproute2-3.2.0/lib/ll_proto.c
--- iproute2-3.2.0.orig/lib/ll_proto.c	2012-01-05 18:34:31.000000000 +0200
+++ iproute2-3.2.0/lib/ll_proto.c	2012-01-06 17:20:34.286187750 +0200
@@ -28,6 +28,10 @@
 #include "utils.h"
 #include "rt_names.h"
 
+/* missing define in PPC if_ether.h */
+#ifndef ETH_P_ECHO
+#define ETH_P_ECHO     0x0200
+#endif
 
 #define __PF(f,n) { ETH_P_##f, #n },
 static const struct {
